cloud/src/meta-service/meta_service_partition.cpp

```diff
@@ -673,6 +677,10 @@ void MetaServiceImpl::drop_partition(::google::protobuf::RpcController* controll
             return;
         }
         versioned_put(txn.get(), operation_log_key, operation_log_value);
+        LOG(INFO) << "put drop partition operation log"
+                  << " instance_id=" << instance_id << " table_id=" << request->table_id()
+                  << " partition_ids=" << drop_partition_log.partition_ids_size()
```

Comment:
`partition_ids` should be `partition_ids_size`? or just all ids?

Reply:
suggest print all partition ids

Reply:
Will fix it in the later PR